libs/community/langchain_community/vectorstores/lancedb.py

```diff
@@ -162,6 +162,7 @@ def get_table(self, name: Optional[str] = None) -> Any:
 
     def create_index(
         self,
+        metric: Optional[str] = "L2",
```

Comment:
This is a breaking change because users may be invoking this code using positional arguments. 

Please put the argument at end and precede it with a `*` operator.

Reply:
Adding it as a `*arg `limits clarity and usage as our user can only specify the arg as a simple string without being able to pass it as a `kwarg (metric = 'cosine' )`, I just shifted this argument to the end of precedence. 

Since its an optional argument and is shifted to the end, it doesnt break previous usage, I have tested it on my end and there is no breakage. Please have a look, Thank you